src/content/docs/en/release/diff-fedora.mdx

```diff
@@ -15,6 +15,13 @@ Changes listed below are nonexhaustive and cumulative, meaning each section cont
 - 🛳️ This ships existing Fedora packages
 - 🔥 This is a significant breaking change
 
+## Ultramarine 41
+
+- 🔥 Switched to Readymade from Anaconda
+- 🔥 Switched to Raspberry Pi Foundation Kernel
+- WSL Support
+- 📦 Chromebook Images
```

Comment:
```suggestion
- 📦 Chromebook Images
- 🛳️ `kcalc` in Plasma Edition instead of `kalk`
- 📦 `mesa-va-drivers-freeworld`, `mesa-vdpau-drivers-freeworld`
- 📦 [`umcli`](https://github.com/Ultramarine-Linux/um) (A CLI tool for managing an Ultramarine Linux system)
- 🛳️ `fastfetch`, `nmtui`
- 📦 switch to `helium-gtk-theme`, `hydrogen-icon-theme`, `ptyxis` on GNOME.
```

Reply:
we always shipped kcalc, but i wanted to switch to kalk in this release, that's not really a change, other than that good

Reply:
Ah… that's why I didn't find it on my system…

Well people seem to like kcalc more, not sure